.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Associated Jira ticket: # (issue)
```

Comment:
Can we agree on having a full link? Going to a board and copy-pasting the id is slower that one click 😅

Reply:
Though do we really need it in the PR? If the issue ID has the right format and is in the commit message, GitHub should turn it into a link on the commit view. That is at least how I always go to Jira issues from PRs.

Reply:
yes, in PR it is not strictly needed (I also look at commits, not PRs), let's mention only URLs in commits